README.md

```diff
@@ -55,3 +55,12 @@ Click on the canvas to enter fullscreen+pointerlock.  Plug in a VR headset, fire
 Move yor head around, turn from side to side, roll, look up and down, and move around within the front facing position sensor's frustum.  Click this screenshot for video!
 
 [![](screenshots/13.png)](https://vimeo.com/136548758)
+
+ ##Run
+ Install nodejs and npm
+ pm install -g  bower
```

Comment:
:smile: thanks for adding these. these notes have actually changed now. bower and submodules and custom command are not needed.

https://github.com/nickdesaulniers/joshVR#local-development